frontend/packages/console-dynamic-plugin-sdk/src/extensions/console-types.ts

```diff
@@ -305,6 +305,8 @@ export type RowProps<D, R extends any = {}> = {
   activeColumnIDs: Set<string>;
 };
 
+export type OnRowsRendered = (params: any) => void;
```

Comment:
[Duplicated from upstream](https://github.com/patternfly/react-virtualized-extension/blob/0c71d4de5aed2ed27cb83ebbd90a16926024bd8b/packages/module/src/components/Virtualized/VirtualTableBody.tsx#L74) since the upstream interface isn't exported.